dotcom-rendering/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.watcherExclude": {
```

Comment:
Can this be moved up a level, above `dotcom-rendering`, to your `.vscode/settings.json` file? If you don't have one, you can create one using these two files: 
- `.vscode/settings.json.recommended`
- `.vscode/settings.json.required`

Up a level, the  `.vscode/settings.json` file is ignored by git

Reply:
I haven't seen it in my commit so my bad! Addressed in aa0367810f0f3dada125612cb1c53b228f3bd821